db/models.py

```diff
@@ -50,3 +53,72 @@
 
     def __str__(self) -> str:
         return f"{self.movie.title} {str(self.show_time)}"
+
+
+class Order(models.Model):
+    created_at = models.DateTimeField(auto_now_add=True)
+    user = models.ForeignKey(
+        settings.AUTH_USER_MODEL,
```

Comment:
Ensure that the custom `User` model is correctly set as the `AUTH_USER_MODEL` in your Django settings.